src/clom/command.py

```diff
@@ -293,8 +294,10 @@ def _clone(self):
     def __eq__(self, other):
         if isinstance(other, string_types):
             return other == str(self)
+        elif isinstance(other, Operation):
+            return vars(other) == vars(self)
         else:
-            return super(self.__class__, self).__eq__(other)
+            return NotImplemented
```

Comment:
Ah, thanks, never came across that one. Would this result in an exception or just result in False for the comparison?